src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+import { getCCAppointmentMock } from '../../mocks/v0';
+import { mockAppointmentInfo } from '../../mocks/helpers';
+import { renderWithStoreAndRouter } from '../../mocks/setup';
+
+import userEvent from '@testing-library/user-event';
+import { AppointmentList } from '../../../appointment-list';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCancel: true,
+    vaOnlineSchedulingRequests: true,
+    vaOnlineSchedulingPast: true,
+    // eslint-disable-next-line camelcase
+    show_new_schedule_view_appointments_page: true,
+    vaOnlineSchedulingHomepageRefresh: true,
+  },
+};
+
+describe('VAOS <AppointmentsPageV2>', () => {
+  beforeEach(() => mockFetch());
+  afterEach(() => resetFetch());
+
+  it('should navigate to community care appointments detail page', async () => {
+    // CC appointment id from confirmed_cc.json
+    const url = '/cc/var8a4885896a22f88f016a2cb7f5de0062';
+
+    const appointment = getCCAppointmentMock();
+    appointment.id = '8a4885896a22f88f016a2cb7f5de0062';
+    appointment.attributes = {
+      ...appointment.attributes,
+      appointmentRequestId: '8a4885896a22f88f016a2cb7f5de0062',
+      distanceEligibleConfirmed: true,
+      name: { firstName: 'Rick', lastName: 'Katz' },
+      providerPractice: 'My Eye Dr',
+      providerPhone: '(703) 555-1264',
+      address: {
+        street: '123',
+        city: 'Burke',
+        state: 'VA',
+        zipCode: '20151',
+      },
+      instructionsToVeteran: 'Bring your glasses',
+      appointmentTime: '05/20/2021 14:15:00',
+      timeZone: 'UTC',
+    };
+
+    mockAppointmentInfo({
+      va: [],
+      cc: [appointment],
+      requests: [],
+      isHomepageRefresh: true,
+    });
+
+    const screen = renderWithStoreAndRouter(
+      <AppointmentList featureHomepageRefresh />,
+      {
+        initialState,
+      },
+    );
+
+    let detailLinks = await screen.findAllByRole('link', {
+      name: /Detail/i,
+    });
+
+    // Select an appointment details link...
+    let detailLink = detailLinks.find(l => l.getAttribute('href') === url);
+    userEvent.click(detailLink);
+
+    // Verify page content...
+    expect(
+      await screen.findByRole('heading', {
+        level: 1,
+        name: /^Thursday, May 20, 2021/,
+      }),
+    ).to.be.ok;
+
+    expect(screen.getByText(/Community Care/)).to.be.ok;
+    expect(await screen.getByText(/123/)).to.be.ok;
```

Comment:
I think you can remove these `await`s. They won't have an effect with `getByText` (vs `findByText`)